libs/ngx-dsfr/navigation/src/lib/navigation.component.html

```diff
@@ -0,0 +1,99 @@
+<nav *ngIf="navigation" class="fr-nav" id="header-navigation" role="navigation" aria-label="Menu principal">
+  <ul class="fr-nav__list">
+    <ng-container *ngFor="let menuItem of navigation">
+      <li class="fr-nav__item">
+        <ng-container *ngIf="!menuItem.isChildAvailable; else childMenuItem">
+          <a
+            class="fr-nav__link"
+            routerLinkActive="active"
+            ariaCurrentWhenActive="page"
+            [routerLink]="[menuItem.href]"
+            target="_self">
+            {{ menuItem.label }}
+          </a>
+        </ng-container>
+        <ng-template #childMenuItem>
+          <!-- MEGA MENU -->
+          <ng-container *ngIf="menuItem.child?.isMega; else simpleMenu">
+            <button
+              (click)="handleMenu(menuItem.id)"
+              #menuButtonItem
+              class="fr-nav__btn"
+              [attr.aria-current]="isRouterLinkActive(menuItem.href) ? true : undefined"
+              [attr.aria-controls]="menuItem.id">
+              {{ menuItem.label }}
+            </button>
+            <div #dropdownItem *ngIf="menuItem.isChildAvailable" class="fr-collapse fr-mega-menu" [id]="menuItem.id" tabindex="-1">
+              <div class="fr-container fr-container--fluid fr-container-lg">
+                <button (click)="hideActiveDropdown()" class="fr-link--close fr-link" [attr.aria-controls]="menuItem.id">Fermer</button>
+                <div class="fr-grid-row fr-grid-row-lg--gutters">
+                  <div class="fr-col-12 fr-col-lg-8 fr-col-offset-lg-4--right fr-mb-4v">
+                    <div class="fr-mega-menu__leader">
+                      <h4 class="fr-h4 fr-mb-2v">{{ menuItem.child?.title }}</h4>
+                      <p class="fr-hidden fr-displayed-lg">{{ menuItem.child?.description }}</p>
+                      <a class="fr-link fr-fi-arrow-right-line fr-link--icon-right fr-link--align-on-content"
+                        [routerLink]="menuItem.href">
+                        Voir toute la rubrique
+                      </a>
+                    </div>
+                  </div>
+                  <div class="fr-col-12 fr-col-lg-3" *ngFor="let item of menuItem.child?.items">
+                    <h5 class="fr-mega-menu__category">
+                      <a
+                        [routerLink]="[item.href]"
+                        routerLinkActive="active"
+                        ariaCurrentWhenActive="page"
+                        class="fr-nav__link"
+                        target="_self">
+                        {{item.label}}
+                      </a>
+                    </h5>
+                    <ul class="fr-mega-menu__list">
+                      <li *ngFor="let link of item.categories">
```

Comment:
It is very weird to find the word `categories` here and not on line 40.

With the suggestions I've made on the `NavigationItem` interface, you could then have: 
```
<li *ngFor="let link of item.children">
```